core-bundle/contao/elements/ContentProxy.php

```diff
@@ -76,17 +84,17 @@ public function generate()
 
 	public function __set($strKey, $varValue)
 	{
-		$this->reference->attributes['templateProperties'][$strKey] = $varValue;
+		$this->reference->attributes['templateProperties'][$strKey] = $this->arrData[$strKey] = $varValue;
```

Comment:
Shouldn't that leave the model data in `arrData` as is? The `templateProperties` have precedence anyways.